DEVEL.md

```diff
@@ -0,0 +1,8 @@
+How to test
+------------------------------
+
+```
+pip install -e ./
+cd tests
+python Test.py
```

Comment:
A comment on intervals: I could imagine cases where it may be important to differentiate open intervals from closed intervals, e.g. [0, 1] or (0, 1]. If the value appears in a denominator or log, this matters. Just a thought but maybe worth further consideration?

Reply:
It is indeed a good point. I'm afraid this would make the implementation much more complicated. I will think a about it. Any suggestion is welcome.
One solution that come to my mind is to let *options* to be "legal" or "illegal" individually and not collectively. This way one can have a simple method for adding open or closed intervals and internally an intervel+option is added.

Reply:
Is `pytest` an additional package then?

Reply:
yes, pytest is a package. It is `pip install`ed in the CI script (*.github/workflows/ci.yml*). it's not mandatory though and not included in *requirements.txt*.

Reply:
Ok so here's my proposal:

How to test
------------------------------

Minimally needed:
```
pip install -e ./
cd tests
python Test.py
```

Recommended:
```
pip install --user pytest
pip install -e ./
cd tests
pytest Test.py
```

Let me know if you don't like it.

Reply:
looks good.